doc/examples/setup_assistant/setup_assistant_tutorial.rst

```diff
@@ -232,97 +280,169 @@ some special operations to happen on them internally.
+`ros2_control <https://control.ros.org/master/index.html>`_.
 
-The 3D Perception tab is meant to set the parameters of the YAML configuration file
-for configuring the 3D sensors **sensors_3d.yaml**.
+This modification adds tags for command and state interfaces for each joint in the defined move groups. 
+The command_interface tags define the types of commands that can be sent to control the joint.
+The state_interface tags define the types of state information that can be read from the joint.
 
-e.g. **point_cloud** parameters:
+By default the Setup Assistant assumes **position** command interface 
+and **position** and **velocity** state interfaces, and we will proceed with this setting. 
 
-.. image:: setup_assistant_panda_3d_perception_point_cloud.png
+.. image:: ros2_control/setup_assistant_ros2_control_tags.png
    :width: 700px
 
-For more details about those parameters please see perception pipeline :doc:`tutorial </doc/examples/perception_pipeline/perception_pipeline_tutorial>`
+If necessary, select the desired command or state interfaces for your robot joints and 
+then click the **Add Interface** button.
+
+Step 9: ROS 2 Controllers
+-------------------------
 
-In case of **sensors_3d.yaml** was not needed, choose **None**.
+The ROS 2 control is a framework for (real-time) control of robots using. 
+ROS 2 Control is designed to manage and simplify the integration of new robot hardware. 
+For more details please look at `ros2_control <https://control.ros.org/master/index.html>`_ documentation.
 
-.. image:: setup_assistant_panda_3d_perception.png
+ROS 2 Controllers tab can be used to auto generate simulated controllers to actuate the robot joints.
+
+.. image:: ros2_controllers/setup_assistant_ros2_controllers.png
    :width: 700px
 
+Add the arm controllers
 
-Step 9: Gazebo Simulation
---------------------------
+* Click on the *ROS 2 Controllers* pane selector.
 
-The Simulation tab can be used to help you simulate your robot with Gazebo by generating a new Gazebo compatible urdf if needed.
+* Click on *Add Controller* and you should see the following screen:
 
-.. image:: setup_assistant_panda_simulation_screen.png
+* We will first add Panda arm joint trajectory controller
+
+* Enter *Controller Name* as **panda_arm_controller**
+
+* Choose **joint_trajectory_controller/JointTrajectoryController** as the controller type
+
+.. image:: ros2_controllers/setup_assistant_panda_arm_ros2_controller_type.png
    :width: 700px
 
-You can use the generated robot **urdf** to spawn the robot in Gazebo in the following way.
+* Next, we need to choose this controller joints. Joints can be added individually or by move group.
 
-* Use rosrun to start gazebo empty world: ::
+* Now, click on Add Planning Group Joints.
 
-   roslaunch gazebo_ros empty_world.launch paused:=true use_sim_time:=false gui:=true throttled:=false recording:=false debug:=true
+* Choose panda_arm planning group to add all the joints in that group to the arm controller.
```

Comment:
Code font for `panda_arm`

Reply:
Choose the `panda_arm` group from the `Available Groups` tab and add it to the `Selected Groups`.

Reply:
Thanks you for your comments @sea-bass and @MarqRazz.

For consistency with using bold font with GUI menus/items, I made it as follows.

Choose the `panda_arm` from the **Available Groups** and add it to the **Selected Groups**.